docs/get-started/setup.md

```diff
@@ -1,3 +1,109 @@
+
+
+#### Build configuration like webpack and Babel
+
+<div style="background-color:#F8FAFC">
+TODO: ask dom/tom presets item points to the preset gallery???
+</div>
+
+If you see errors on the CLI when you run the `yarn storybook` command. It’s likely you need to make changes to Storybook’s build configuration. Here are some things to try:
+
+- [Presets](../presets/preset-gallery/) bundle common configurations for various technologies into Storybook. In particular presets exist for Create React App, SCSS and Ant Design.
+- Specify a custom [Babel configuration](../configure/integration#custom-babel-config) for Storybook. Storybook automatically tries to use your project’s config if it can.
+- Adjust the [webpack configuration](../configure/integration#Webpack) that Storybook uses. Try patching in your own configuration if needed.
+
+#### Runtime configuration
+
+If Storybook builds but you see an error immediately when connecting to it in the browser, then chances are one of your input files is not compiling/transpiling correctly to be interpreted by the browser. Storybook supports modern browsers and IE11, but you may need to check the Babel and webpack settings (see above) to ensure your component code works correctly.
+
+
+<h4 id="story-context" name="story-context">Story context</h4>
+
+If a particular story has a problem rendering, often it means your component expects a certain environment is available to the component. 
+
+A common frontend pattern is for components to assume that they render in a certain “context” with parent components higher up the rendering hierarchy (for instance theme providers)
+
+Use [decorators](../writing-stories/decorators) to “wrap” every story in the necessary context providers. [`.storybook/preview.js`](../writing-stories/decorators#global-decorators) allows you to customize how components render in Canvas, the preview iframe. In this decorator example, we wrap every component rendered in Storybook with `ThemeProvider`
+
+```js
+// .storybook/preview.js
+
+import { ThemeProvider } from ‘styled-components’;
+export const decorators = [
+  (Story) => <ThemeProvider theme=”default”><Story/></ThemeProvider>,
+];
+
+```
+
+### Render component styles
+
+Storybook isn’t opinionated about how you generate or load CSS. It renders whatever DOM elements you provide. But sometimes things won’t “look right” out of the box. 
+
+You may have to configure your CSS tooling for Storybook’s rendering environment. Here are some tips on what could help:
+
+
+#### CSS-in-JS like styled components and emotion
+
+If you are using CSS-in-JS, chances are your styles are working because they’re generated in JavaScript and served alongside each component. 
+
+Theme users may need to add a decorator to `.storybook/preview.js`, [see above](#story-context).
+
+
+#### @import CSS into components
+
+<div style="background-color:#F8FAFC">
+TODO:ask tom/dom for angular special import
```

Comment:
This thing: https://storybook.js.org/docs/configurations/default-config/#css-support

Reply:
I'm not sure where we should put it, maybe in configuration somewhere

Reply:
My thougths exactly. If we start adding edge cases we start to loose focus. This document should tell readers how they setup Storybook. I've updated the link to where it could point. 

Also took a note about this and once i get there i'll address this particular case. It will need a bit of change in rewording probably.